tailwind.config.cjs

```diff
@@ -1,10 +1,9 @@
-const defaultOptions = require('../traction-ui/node_modules/@sanger/ui-styling/tailwind.config')
+const defaultOptions = require('@sanger/ui-styling/tailwind.config')
+
 module.exports = {
   ...defaultOptions,
-  purge: {
-    ...defaultOptions.purge,
-    content: ['./src/**/*.{html,vue}'],
-  },
+
+  content: [...defaultOptions.content, './src/**/*.{html,vue}'],
```

Comment:
Content is defined already, I'm assuming its passed through from tailwind itself.

Reply:
> Content is defined already, I'm assuming its passed through from tailwind itself.

 Looking good. 'purge' is not in use any more. In ui-styling package , content supports html and react(jsx,tsx) files. So, I think over riding  here for vue files is required.